infrastructure/infra-aws.tf

```diff
@@ -251,24 +251,10 @@ resource "aws_iam_role_policy" "kinesis-firehose-es" {
             "Action": "logs:CreateLogGroup",
             "Resource": "*"
         },
-        {
-            "Effect": "Allow",
-            "Action": "es:*",
-            "Resource": "arn:aws:es:*:*:*"
-        },
         {
             "Effect": "Allow",
             "Action": "s3:*",
             "Resource": "arn:aws:s3:::*"
```

Comment:
Can we limit s3 access to the specific bucket that the processor needs to access?